ideas/172-topic-democracy.md

```diff
@@ -0,0 +1,83 @@
+# Democracy
```

Comment:
Could you add yaml frontmatter a la e.g. https://raw.githubusercontent.com/status-im/ideas/master/ideas/58-mainnet.md?

This way it will show up at https://ideas.status.im/

Reply:
Done.